bin/jposts

```diff
@@ -8,32 +8,36 @@ var defaultHeader = {layout: 'post',title: '',tags: '',categories: ''}
 
 program
   .version(VERSION)
-  // .option('-d, --date', 'Get current Date(format dd/mm/yyyy)')  
-  // .option('-t, --tags', 'Set the current article tags')  
+  // .option('-d, --date', 'Get current Date(format dd/mm/yyyy)')
+  // .option('-t, --tags', 'Set the current article tags')
 
 program
   .command('create-post <postTitle>')
   .description('Create a post')
   .option('-c, --categories <items>', 'Specify the categor(y/ies) for the current article \n                          example: jposts create <postName> -c category1,category2,...')
+  .option('-t, --tags <items>', 'Specify the categor(y/ies) for the current article \n                          example: jposts create <postName> -t tag1,tag2,...')
   .option('-l, --layout <name>', "If set, this specifies the layout file to use. Use the layout file\n                          name without the file extension. Layout files must be placed \n                          in the  '_layouts' directory.")
   .action(function (postTitle, options) {
     'use strict'
 
     var originalPostTitle = postTitle.trim().replace(/[\s]{2,}/, '').replace(/[ ]/g, '-')
     var finalPostTitle = utilities.getCurrentDate().join('-') + '-' + originalPostTitle + '.md'
     var filename = '.' + path.sep + '_posts' + path.sep + finalPostTitle
-    // check if the title is not empty 
+    // check if the title is not empty
     if (postTitle) {
-      // check if the article doesn't exist already          
+      // check if the article doesn't exist already
       if (utilities.fileExists(filename)) {
         console.log('file already exists ')
       } else {
         defaultHeader.title = postTitle
         // console.log('file does not exist')
-         
+
         if (options.categories) {
           defaultHeader.categories = options.categories.split(',')
         }
+        if (options.tags) {
+          defaultHeader.tags = options.tags.split(',')
```

Comment:
Please, change `options.tags.split(',')` to `options.tags.replace(/[\s]{1,}/g, '').split(',')` so tags with white spaces will be rendered correctly.